src/Particle/SoaDistanceTableAA.h

```diff
@@ -25,8 +25,8 @@ struct SoaDistanceTableAA : public DTD_BConds<T, D, SC>, public DistanceTableDat
   ///number of targets with padding
   int Ntargets_padded;
 
-  ///actual memory for displacements_
-  aligned_vector<RealType> memory_pool_displs_;
+  ///actual memory for dist and displacements_
+  aligned_vector<RealType> memory_pool_;
```

Comment:
Are RealType and T the same? somtimes/alaways  I gave up trying to figure that out.

Reply:
I will figure that out.

Reply:
Probably some cleaning will be need both in the base also in this class.

Reply:
Perhaps an issue makes sense?